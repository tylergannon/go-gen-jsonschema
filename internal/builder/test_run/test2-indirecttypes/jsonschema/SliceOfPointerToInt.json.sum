61700f20c05dbe7c