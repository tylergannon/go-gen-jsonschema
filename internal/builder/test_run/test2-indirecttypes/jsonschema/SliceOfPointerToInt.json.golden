{
  "type": "array",
  "description": "SliceOfPointerToInt is a slice of pointers to int",
  "items": {
    "type": "integer"
  }
}
